### DISCUSSION TITLE
Hope sinytra team transplant (my personal opinion)

### DISCUSSION THREAD

**Discussion by 323237216** [Sinytra/Connector#533] [Polls]
Do you feel that the Sinytra team is too focused on the 1.20.1 Minecraft version and many mods are not 1.20.1 versions?

**Comment by Su5eD** [comment]
As previously said, no backports are planned. Our focus is on future versions, where we can make the necessary changes to simply connector's code. In the end, all of Sinytra's projects are open source, and anyone is free to make a backport should they want to.

**Reply by 323237216** [reply]
Minecraft version 1.20.2 is the latest official release, and after 1.20.1, is a more future version

**Reply by coco875** [reply]
he want to update for 1.20.2 but for now neoforge don't release a stable version so he will not port until a release of neoforge not forge

**Reply by 323237216** [reply]
OK
